plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -16,35 +17,85 @@ class QueueType(Enum):
 class SlurmQueueConf:
     # Params are used to configure sbatch, for more info check:
     # https://github.com/facebookincubator/submitit/blob/master/submitit/slurm/slurm.py
-    nodes: int = 1
+
+    # maximum time for the job in minutes
+    time: int = 60
+    # number of cpus to use for each task
+    cpus_per_task: int = 10
+    # number of gpus to use on each node
     gpus_per_node: int = 1
+    # number of tasks to spawn on each node
     ntasks_per_node: int = 1
+    # number of nodes to use for the job
+    nodes: int = 1
+    # memory to reserve for the job on each node
     mem: str = "${hydra.launcher.mem_limit}GB"
-    cpus_per_task: int = 10
-    time: int = 60
+    # slurm partition to use on the cluster
+    # (depending on the cluster configuration, default "dev" may not exist)
```

Comment:
Is there a choice that always works? what happens if you don't specify the partition?

We can get a different default on the FAIR cluster (or other environments quiet easily through a ConfigSearchPathPlugin).

Reply:
> Is there a choice that always works? what happens if you don't specify the partition?

Good point, I though that there was a default value in submitit but there is none, so it falls back to the default partition of the cluster, better initialize it as None then